TransformManager.cs

```diff
@@ -103,6 +103,18 @@ public void RegisterGizmoType(string sType, ITransformGizmoBuilder builder)
             _gizmoTypes[sType] = builder;
         }
 
+        /// <summary>
+        /// Associate a new gizmo builder with an identifier
```

Comment:
We un-register gizmo but the comment says "associate", probably a 'copy-paste'.

Reply:
changed